### DISCUSSION TITLE
Can you use backticks ` in a fenced code block with a titile?

### DISCUSSION THREAD

**Discussion by kenny-chua** [squidfunk/mkdocs-material#7402] [Q&A] [answered]
EDIT: look at the raw text to see the clearer question, GH is rendering the markdown syntax
example:

``` py title="My name is `Bob`"
code
```

I can't seem to escape it properly. Logically, I would do:

``` py title="My name is \`Bob\`"
code
```

Am I missing something?

**Comment by facelessuser** [comment, marked as answer]
I'm a little unclear on exactly what you are asking, so correct me if I have misinterpreted your question, but hhy not just do this?

````
```py title="My name is `Bob`"
import code
```
````

<img width="362" alt="Screenshot 2024-07-28 at 10 49 56 AM" src="https://github.com/user-attachments/assets/253f0e04-b44f-4951-878c-afd413cb81d0">

**Reply by facelessuser** [reply]
It should be noted that the title attribute is not parsed as Markdown in SuperFences, so no escaping is needed.

**Reply by kenny-chua** [reply]
MY BAD!!!! That's what I thought. It's my linter in VS Code that tricked me that something is wrong!! I'm so sorry!!

squiggly line error
<img width="311" alt="image" src="https://github.com/user-attachments/assets/ee381774-5496-4832-849b-b1e1417bc3f6">

RENDER OK!!!
<img width="242" alt="image" src="https://github.com/user-attachments/assets/0806e5d3-d514-4c92-8c87-f43b771624de">

I apologize for the confusion. I depend too much on linters. I need to figure out how to turn that off!! It's coming from extension "markdownlint". Gotta find that rule and turn it off!!

The linting rule that is applied is that "MD040 - Fenced code blocks should have a language specified", but I'm suspecting it doesn't differentiate between the ` and "

Thanks again, and I apologize!!

**Reply by kenny-chua** [reply]
you don't happen to know what rule I should turn off do you? if you use markdown lint? It's very strict, but I like it because it makes the document in markdown consistent, do you use one or have a different suggestion?

**Reply by facelessuser** [reply]
To be honest, I do not as I don't use that linter, sorry.

**Reply by kenny-chua** [reply]
do you use any linter for markdown?

**Reply by facelessuser** [reply]
No, not really. I'm pretty consistent in my writing patterns and structures...now spell checking on the other hand, I misspell things all the time, so I enable stuff like that.

If I was managing a project with a ton of contributions to my documentation, I could see that possibly pushing me to enable at least some Markdown linting to enforce consistency. It seems most people hate writing documentation and would rather contribute code or fix small typos 🙃.

**Reply by kenny-chua** [reply]
I know, and I'm the perfectionist, who spends more time in beautifying documentation rather than coding. lol. But thanks again!!
